### DISCUSSION TITLE
Allow issues in private repositories to be publically viewable

### DISCUSSION THREAD

**Discussion by luxaritas** [community/community#5645] [Projects and Issues]
It would be great for our project to be able to provide transparency in what we're working on while keeping our code private. This would also mean that they can be shown in detail in public projects. While we could create a separate public repo just for issues, it is preferable to keep them in the same repo that they apply to.

A potential limitation here is that there may be some issues that contain information that should not be public - it could be useful to allow for individual issues to be converted to public or private, though issues on the entire repo may default to one or the other.

Related: #4271, #5415, #4381

**Comment by luxaritas** [comment]
Additional addendum: It could be useful to get even more fine-grained than this. Eg, on a per-repo and per-issue basis, decide what information to expose (title, description, comments - or even specific comments, ...)

**Comment by SugarD-x** [comment]
This is something I have been pushing for throughout the years myself. I can't find the old issue discussion anymore because it appears GitHub hid everything when they created the "Discussions" section under their Community repo, but many users supported this at the time too. The fact that other git-based repo systems have this as a regular option and GitHub doesn't is absolutely mind-blowing. Especially when many closed-source projects hosted here are publicly available in other forms, (including as compiled versions), and desperately need to have user feedback for issues that are found.
